src/learn.jl

```diff
@@ -63,6 +63,16 @@ Log a value `value` to `field`.
 """
 log_value!(logger, field::AbstractString, value)
 
+
+"""
+    summarize_array(logger::Any, v)
+
+Summarize an array `v`, returning a value `x` supported by `Lighthouse.log_value!(logger, x)`.
+
+Defaults to taking the `mean`.
+"""
+summarize_array(logger::Any, v) = mean(v)
```

Comment:
I think I'm missing some key piece of why this needs to be dispatched on a logger in the first place---any chance you could link me (via slack!) to our private repo usage of this, or copy in a MWE?

Reply:
I’ll link you my usage on slack, but the reason is because I want this to be a `Wandb.Histogram` for a (not yet open source) WandbLogger, but I don’t want to depend on wandb here. And I want to use it in LighthouseFlux that also doesn’t know about wandb.

My thought is it kinda makes sense generally bc the best way to summarize an array might depend on the capabilities of the logger. Eg maybe some loggers would like to save a heatmap or something like that. Wandb supports histograms really well so that makes sense for that logger. Etc. And  `mean` gives a fairly safe fallback.

Reply:
Discussed on Slack that a better approach is `log_array!`.